src/api/PDFDocument.ts

```diff
@@ -713,6 +713,29 @@ export default class PDFDocument {
     return copiedPages;
   }
 
+  /**
+   * Add document JavaScript. The script is executed when the document is opened.
+   * See the [JavaScript™ for Acrobat® API Reference](https://www.adobe.com/content/dam/acom/en/devnet/acrobat/pdfs/js_api_reference.pdf)
+   * for details.
+   * @param script The JavaScript to execute.
+   * @param name The name of the script. Must be unique per document.
+   */
```

Comment:
The standard for the pdf-lib codebase is to add example code to all doc comments for each public method. Please add one here. You can reference other methods to see what this should look like.

Reply:
I added a short example.